Tiltfile

```diff
@@ -183,31 +183,31 @@ def launch_node_year_service(auto_init=True):
   launch_node_svc("year-node", dirname="node/year-service", auto_init=auto_init)
 
 # Launch one of each of these types of services. Go services init by default
-launch_go_frontend()
+# launch_go_frontend()
```

Comment:
We usually don't commit the launch diffs, and keep it defaulting to Go

Reply:
Ooops! Sorry about that.